web_set_single_page_hidden/static/src/js/pager.js

```diff
@@ -0,0 +1,14 @@
+odoo.define('web_set_single_page_hidden.Pager', function (require) {
+    "use strict";
+
+    var Pager = require('web.Pager');
+
+    Pager.include({
+        set_single_page_hidden: function (value) {
+            if (!_.isUndefined(this.options)) {
```

Comment:
why use _.isUndefined ? 

```suggestion
            if (this.options) {
```

Reply:
Indeed, your suggestion has better readability. I'll change it :)